sycl/test-e2e/bindless_images/cubemap/cubemap_sampled.cpp

```diff
@@ -0,0 +1,143 @@
+// REQUIRES: linux
+// REQUIRES: cuda
+// REQUIRES: aspect-ext_oneapi_cubemap
+// REQUIRES: aspect-ext_oneapi_cubemap_seamless_filtering
```

Comment:
Do multiple `REQUIRES` line work as `&&`? Have you made sure the test is actually running?

Reply:
I can't combine the `REQUIRES` lines with `&&` but a simple `,` seems to work. I've switched to that where it fits in a line. Yes the test is running :smile: